service/docs/source/geopmagent.1.rst

```diff
@@ -40,16 +44,16 @@ Options
 -a AGENT    Specify the name of the agent.
 -p POLICY   The values to be set for each policy in a comma-separated list.
             Values other than the first policy are optional and will be set to
-            NAN if not provided, indicating that the Agent should use a default
+            ``NAN`` if not provided, indicating that the Agent should use a default
             value.  If the agent does not require any policy values this option
-            must be specified as "None" or "none".
+            must be specified as ``"None"`` or ``"none"``.
 
 Examples
 --------
 
 List all available agents on the system:
 
-.. code-block::
+.. code-block:: bash
```

Comment:
Maybe use `console` instead of `bash` since this block has a mix of input (prefixed with `$`) and command output.